examples/web/index.html

```diff
@@ -69,6 +69,56 @@ <h2>Tag</h2>
           <span class="Tag Tag--danger-dark">I'm tag. I feel like filling marketing pages :-)</span>
         </div>
       </section>
+
+      <section>
+        <h2>TextField</h2>
+        <div style="max-width: 25rem">
```

Comment:
I'd upgrade the examples a bit:

* wrap section content with `Container`
* wrap the fields with an example layout like this:

   ```css
   display: grid;
   grid-template-columns: 25rem;
   gap: 1.5rem;
   ```